### DISCUSSION TITLE
I see a number on the display but not the word PAX in front of it.

### DISCUSSION THREAD

**Discussion by hjojohan** [cyberman54/ESP32-Paxcounter#962] [General]
Hi 
I am a newbie and just downloaded the source code and platform.io. I can build and upload the code to the TTGO V21NEW chip 
and I used the plain decoder.  In my IOT cloud the device joins just fine and report data but I cant see the actual mobile phone count. 

Only rssi , wifi and timesync_seqno are visible in the payload.. Does anyone understand what I have configured wrong? 

![pax1](https://github.com/cyberman54/ESP32-Paxcounter/assets/6337514/5393f40e-a410-4694-bf68-60e6ef4917b0)

**Comment by cyberman54** [comment]
please show your paxcounter.conf file.
"pax" is just sum of "wifi" and "ble". If you don't see "ble", you probably did not configure ble scanning (either not compiled in code, or switched off).

**Comment by hjojohan** [comment]
Please do a quick look in my config file to se if everything looks right..  Can the device connect to my local Wifi and send data directly to my MQTT broker?  What is the aproximative range the device will detect devices? 

[paxcounter.txt](https://github.com/cyberman54/ESP32-Paxcounter/files/11488755/paxcounter.txt)
